### DISCUSSION TITLE
MA 2.4.0b8, playing via Airplay or Sonos S2- physical remote control locked out?

### DISCUSSION THREAD

**Discussion by mkammes** [music-assistant/support#3379] [Q&A]
Hey folks:

I'm encountering an issue, and I cannot determine whether it's by design.

MA 2.4.0b8 running within Home Assistant, but I'm *not* using the HA integration.

My MA sources (3 of them: Radio/Sirius XM, Playlist/Spotify, Artist/Band/Plex....I'm not using DLNA/UPnP or others) sent to Sonos S2 speakers or via Airplay to the same S2 speakers is played with no problem. However, I lose all volume control via physical remotes. As my S2 speakers are connected via eARC to the Television, I can see the activity/notification LED on the television react to my key presses on the physical remotes, but, no audio change happens (volume up/down/mute).

I've seen some forum posts that possibly allude to [Flow Mode needing to be disabled if local control is lost/not available](https://music-assistant.io/player-support/sonos/) however, it appears the GUI toggle was removed.

Is this activity expected?

Thanks!

**Comment by OzGav** [comment]
Good question. @marcelveldt ?

**Reply by marcelveldt** [reply]
I have no idea, probably not related to MA at all. I'll double check on my end tomorrow but as far as I know all volume controls work.

**Comment by OzGav** [comment]
Just trying to follow up here. Are you trying to use the TV remote to change the volume on the Sonos speakers (which are connected via ARC)? If so that is unlikely to work.
